contrib/grimshot

```diff
@@ -91,12 +111,12 @@ takeScreenshot() {
   FILE=$1
   GEOM=$2
   OUTPUT=$3
-  if [ ! -z "$OUTPUT" ]; then
-    grim -o "$OUTPUT" "$FILE" || die "Unable to invoke grim"
+  if [ -n "$OUTPUT" ]; then
+    grim ${CURSOR:+-c} -o "$OUTPUT" "$FILE" || die "Unable to invoke grim"
```

Comment:
... And then just using `$CURSOR` here?

I believe this syntax here is the only bashism, right?

Reply:
As I wrote above bash is used because of the array, so this expansion is not *the* bashism or the most significant one; actually I'm not sure if it's a bashism

Reply:
Well, it doesn't seem to be a bashism — at least it's supported on `dash` and `shellcheck` doesn't report it as an extension